starboard/android/shared/video_render_algorithm.cc

```diff
@@ -107,8 +107,10 @@ void VideoRenderAlgorithm::Render(
       frames->pop_front();
       ++dropped_frames_;
     } else if (early_us < kBufferReadyThreshold) {
-      auto status = draw_frame_cb(frames->front(), adjusted_release_time_ns);
-      SB_DCHECK(status == VideoRendererSink::kReleased);
+      if (!video_decoder_->IsTunnelMode()) {
```

Comment:
It seems to me that what the algorithm for non-tunneled mode does is:
1. If the frame arrives too late (16 or 32 ms later than the time it should be displayed), we drop it.
2. Else if the frame should be rendered in the next 50 ms, we display it.
3. Else do nothing.

What do we want for tunnel mode frames?  Will we get the same accuracy after tick interval is reduced to 10ms?